frontend/lit/components/Reference.js

```diff
@@ -114,7 +114,7 @@ class Reference extends Component {
                 <ActionLink key={0} label="Edit reference tags" href={data.editTagUrl} />,
                 <ActionLink key={1} label="Edit reference" href={data.editReferenceUrl} />,
                 <ActionLink key={2} label="Delete reference" href={data.deleteReferenceUrl} />,
-                <ActionLink key={3} label="Tag history" href={data.tagHistoryUrl} />,
+                <ActionLink key={3} label="Tag state" href={data.tagStateUrl} />,
```

Comment:
Technically, I think you're correct that this is the tag state. However, for our users, generally the term state may be confusing as it's a concept they're likely not very familiar with. Consider changing back to "Tag history" or "User tags"